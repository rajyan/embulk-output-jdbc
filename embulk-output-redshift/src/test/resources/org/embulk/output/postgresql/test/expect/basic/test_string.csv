id:long,char_item:string,varchar_item:string,text_item:string
1,a,b,c
2,abc,def,ABCDEFGHIJ
3,,,
