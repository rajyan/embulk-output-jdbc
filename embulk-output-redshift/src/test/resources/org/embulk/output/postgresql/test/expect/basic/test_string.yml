table: test_string
mode: insert
