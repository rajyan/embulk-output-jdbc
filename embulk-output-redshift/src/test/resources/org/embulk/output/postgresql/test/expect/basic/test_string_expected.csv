1,a   ,b,c
2,abc ,def,ABCDEFGHIJ
3,\N,\N,\N
